pygmt/datasets/load_earth_dataset.py

```diff
@@ -0,0 +1,210 @@
+"""
+Internal function to load remote Earth datasets.
```

Comment:
It's likely that GMT will add more datasets for Moon or Mars. I believe the same load function can be used for these non-Earth datasets. 
```suggestion
Internal function to load GMT remote datasets.
```